parachain/pallets/inbound-queue/src/mock.rs

```diff
@@ -219,6 +219,8 @@ impl inbound_queue::Config for Test {
 	#[cfg(feature = "runtime-benchmarks")]
 	type Helper = Test;
 	type WeightToFee = IdentityFee<u128>;
+	type LengthToFee = IdentityFee<u128>;
+	type MaxMessagePayloadSize = ConstU32<1024>;
```

Comment:
The size is the payload as well as the proof, and according to my guess its going to be around 2KIB,

```suggestion
	type MaxMessageSize = ConstU32<2048>;
```

Reply:
This is the mock and yeah 2048 is the value in runtime.

Reply:
Rename to `MaxMessageSize` please

Reply:
https://github.com/Snowfork/snowbridge/pull/1065/commits/1033090342e61c1d5a86a893187794c971e553f1